runtime/lua/vim/treesitter/languagetree.lua

```diff
@@ -754,6 +762,12 @@ function LanguageTree:_get_injection(match, metadata)
   local combined = metadata['injection.combined'] ~= nil
   local lang = metadata['injection.language'] --[[@as string?]]
   local include_children = metadata['injection.include-children'] ~= nil
+  if metadata['injection.self'] ~= nil then
```

Comment:
Shouldn't `self/parent` only happen if `injection.language` doesn't exist?

Reply:
I just made it the other way round now